### DISCUSSION TITLE
Skip when click the close button

### DISCUSSION THREAD

**Discussion by busyxiang** [gilbarbara/react-joyride#762] [Q&A] [answered]
I want to end the tutorial when the user clicks the close button, but now the action **close** is shared with clicking overlay too, is there a specific callback for clicking the close button?

**Comment by gilbarbara** [comment, marked as answer]
If you want to control the tour state you probably need to use the controlled mode

**Reply by busyxiang** [reply]
@gilbarbara thank you for the advice, in the end I chose to use custom tooltip component, and pass skipProps to the close button
